uitest task broken after gradle upgrade

Last one failing due to test failing, first one failing after gradle upgrade: https://treeherder.mozilla.org/#/jobs?repo=reference-browser&group_state=expanded&resultStatus=testfailed%2Cbusted%2Cexception%2Cretry%2Cusercancel%2Crunnable&tochange=7bd1d8b745575df1cbeecf43f64b8d2a05ac81cd&fromchange=176202b13cb2006ca2ddb6ce668807479d1f8b4f&selectedJob=292312415

Log: https://treeherder.mozilla.org/logviewer.html#?job_id=292312415&repo=reference-browser

[task 2020-03-19T17:27:49.483Z] > Task :app:compileDebugKotlin
[task 2020-03-19T17:27:49.483Z] w: /builds/worker/checkouts/src/app/src/main/java/org/mozilla/reference/browser/BrowserApplication.kt: (87, 44): 'onLowMemory(): Unit' is deprecated. Use onTrimMemory instead.
[task 2020-03-19T17:27:49.483Z] w: /builds/worker/checkouts/src/app/src/main/java/org/mozilla/reference/browser/components/UseCases.kt: (55, 34): 'constructor WebAppUseCases(Context, SessionManager, Client, Boolean = ...)' is deprecated. Pass in WebAppShortcutManager directly instead
[task 2020-03-19T17:27:49.582Z] 
[task 2020-03-19T17:27:49.582Z] > Task :app:mergeExtDexDebug
[task 2020-03-19T17:27:51.182Z] > Task :app:mergeDebugNativeLibs
[task 2020-03-19T17:27:53.182Z] 
[task 2020-03-19T17:27:53.182Z] > Task :app:stripDebugDebugSymbols
[task 2020-03-19T17:27:53.182Z] WARNING: Compatible side by side NDK version was not found. Default is 20.0.5594570.
[task 2020-03-19T17:27:53.182Z] Compatible side by side NDK version was not found. Default is 20.0.5594570.
[task 2020-03-19T17:27:53.182Z] Unable to strip the following libraries, packaging them as they are: libfreebl3.so, libfreebl3.so, libfreebl3.so, libfreebl3.so, libglean_ffi.so, libglean_ffi.so, libglean_ffi.so, libglean_ffi.so, libjnidispatch.so, libjnidispatch.so, libjnidispatch.so, libjnidispatch.so, libjnidispatch.so, libjnidispatch.so, libjnidispatch.so, liblgpllibs.so, liblgpllibs.so, liblgpllibs.so, liblgpllibs.so, libmegazord.so, libmegazord.so, libmegazord.so, libmegazord.so, libmozavcodec.so, libmozavcodec.so, libmozavcodec.so, libmozavcodec.so, libmozavutil.so, libmozavutil.so, libmozavutil.so, libmozavutil.so, libmozglue.so, libmozglue.so, libmozglue.so, libmozglue.so, libnss3.so, libnss3.so, libnss3.so, libnss3.so, libnssckbi.so, libnssckbi.so, libnssckbi.so, libnssckbi.so, libplugin-container.so, libplugin-container.so, libplugin-container.so, libplugin-container.so, libsoftokn3.so, libsoftokn3.so, libsoftokn3.so, libsoftokn3.so, libxul.so, libxul.so, libxul.so, libxul.so.
[task 2020-03-19T17:27:53.282Z] 
[task 2020-03-19T17:27:53.282Z] > Task :app:mergeDebugAndroidTestAssets
[task 2020-03-19T17:27:53.382Z] > Task :app:javaPreCompileDebug
[task 2020-03-19T17:27:57.682Z] > Task :app:compileDebugJavaWithJavac
[task 2020-03-19T17:27:57.684Z] > Task :app:compileDebugSources
[task 2020-03-19T17:27:59.482Z] > Task :app:dexBuilderDebug
[task 2020-03-19T17:27:59.482Z] > Task :app:mergeDebugAndroidTestNativeLibs
[task 2020-03-19T17:27:59.582Z] > Task :app:mergeDebugAndroidTestResources
[task 2020-03-19T17:27:59.682Z] > Task :app:mergeDebugJavaResource
[task 2020-03-19T17:27:59.982Z] > Task :app:desugarDebugAndroidTestFileDependencies
[task 2020-03-19T17:27:59.982Z] > Task :app:bundleDebugClasses
[task 2020-03-19T17:28:00.282Z] > Task :app:processDebugAndroidTestResources
[task 2020-03-19T17:28:03.682Z] 
[task 2020-03-19T17:28:03.686Z] > Task :app:compileDebugAndroidTestKotlin FAILED
[task 2020-03-19T17:28:03.686Z] e: /builds/worker/checkouts/src/app/src/androidTest/java/org/mozilla/reference/browser/helpers/MockWebServer.kt: (15, 42): Unresolved reference: toUri
[task 2020-03-19T17:28:03.687Z] e: /builds/worker/checkouts/src/app/src/androidTest/java/org/mozilla/reference/browser/helpers/MockWebServer.kt: (27, 79): Unresolved reference: toUri
[task 2020-03-19T17:28:03.688Z] e: /builds/worker/checkouts/src/app/src/androidTest/java/org/mozilla/reference/browser/helpers/TestAssetHelper.kt: (9, 42): Unresolved reference: toUri
[task 2020-03-19T17:28:03.689Z] e: /builds/worker/checkouts/src/app/src/androidTest/java/org/mozilla/reference/browser/helpers/TestAssetHelper.kt: (33, 68): Unresolved reference: toUri
[task 2020-03-19T17:28:03.691Z] e: /builds/worker/checkouts/src/app/src/androidTest/java/org/mozilla/reference/browser/helpers/TestAssetHelper.kt: (40, 71): Unresolved reference: toUri
[task 2020-03-19T17:28:03.692Z] e: /builds/worker/checkouts/src/app/src/androidTest/java/org/mozilla/reference/browser/helpers/TestAssetHelper.kt: (47, 67): Unresolved reference: toUri
[task 2020-03-19T17:28:03.693Z] e: /builds/worker/checkouts/src/app/src/androidTest/java/org/mozilla/reference/browser/helpers/TestAssetHelper.kt: (53, 63): Unresolved reference: toUri
[task 2020-03-19T17:28:03.695Z] e: /builds/worker/checkouts/src/app/src/androidTest/java/org/mozilla/reference/browser/helpers/TestAssetHelper.kt: (60, 70): Unresolved reference: toUri
[task 2020-03-19T17:28:03.696Z] 
[task 2020-03-19T17:28:03.697Z] > Task :app:mergeProjectDexDebug
[task 2020-03-19T17:28:03.782Z] > Task :app:mergeExtDexDebugAndroidTest
[task 2020-03-19T17:28:03.882Z] 
[task 2020-03-19T17:28:03.882Z] FAILURE: Build failed with an exception.
[task 2020-03-19T17:28:03.882Z] 
[task 2020-03-19T17:28:03.883Z] * What went wrong:
[task 2020-03-19T17:28:03.883Z] Execution failed for task ':app:compileDebugAndroidTestKotlin'.
[task 2020-03-19T17:28:03.883Z] > Compilation error. See log for more details
[task 2020-03-19T17:28:03.883Z] 
[task 2020-03-19T17:28:03.883Z] * Try:
[task 2020-03-19T17:28:03.883Z] Run with --stacktrace option to

--- a/app/src/androidTest/java/org/mozilla/reference/browser/helpers/MockWebServer.kt
+++ b/app/src/androidTest/java/org/mozilla/reference/browser/helpers/MockWebServer.kt
@@ -12,7 +12,6 @@ import okhttp3.mockwebserver.Dispatcher
 import okhttp3.mockwebserver.MockResponse
 import okhttp3.mockwebserver.MockWebServer
 import okhttp3.mockwebserver.RecordedRequest
-// import org.mozilla.reference.browser.ext.String.toUri
 import org.mozilla.reference.browser.helpers.ext.toUri
 import java.io.IOException
 

--- a/app/src/androidTest/java/org/mozilla/reference/browser/helpers/MockWebServer.kt
+++ b/app/src/androidTest/java/org/mozilla/reference/browser/helpers/MockWebServer.kt
@@ -12,7 +12,8 @@ import okhttp3.mockwebserver.Dispatcher
 import okhttp3.mockwebserver.MockResponse
 import okhttp3.mockwebserver.MockWebServer
 import okhttp3.mockwebserver.RecordedRequest
-import org.mozilla.reference.browser.ext.toUri
+// import org.mozilla.reference.browser.ext.String.toUri
+import org.mozilla.reference.browser.helpers.ext.toUri
 import java.io.IOException
 
 object MockWebServerHelper {

--- a/app/src/androidTest/java/org/mozilla/reference/browser/helpers/TestAssetHelper.kt
+++ b/app/src/androidTest/java/org/mozilla/reference/browser/helpers/TestAssetHelper.kt
@@ -6,7 +6,7 @@ package org.mozilla.reference.browser.helpers
 
 import android.net.Uri
 import okhttp3.mockwebserver.MockWebServer
-import org.mozilla.reference.browser.ext.toUri
+import org.mozilla.reference.browser.helpers.ext.toUri
 import java.util.concurrent.TimeUnit
 
 /**

